src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs

```diff
@@ -161,23 +162,25 @@ protected override void ExecuteCore()
                 ParsePackageAssemblyReferences(BaselinePackageAssemblyReferences));
         }
 
-        private static Dictionary<string, string[]>? ParsePackageAssemblyReferences(ITaskItem[]? packageAssemblyReferences)
```

Comment:
Is it possible to use the `NugetFramework` as the key and avoid having to pass more information, or is it not preserved in   its combined form on the content items?